plugins/backend/catalog-backend-module-okta/src/providers/OktaUserEntityProvider.ts

```diff
@@ -52,13 +54,18 @@ export class OktaUserEntityProvider extends OktaEntityProvider {
     accountConfig: AccountConfig,
     options: {
       logger: winston.Logger;
+      customAttributesToAnnotationAllowlist?: string[];
       namingStrategy?: UserNamingStrategies | UserNamingStrategy;
     },
   ) {
     super([accountConfig], options);
     this.namingStrategy = userNamingStrategyFactory(options.namingStrategy);
     this.userFilter = accountConfig.userFilter;
     this.orgUrl = accountConfig.orgUrl;
+    this.customAttributesToAnnotationAllowlist =
+      options.customAttributesToAnnotationAllowlist
+        ? options.customAttributesToAnnotationAllowlist
+        : [];
```

Comment:
we could write this expression like the below suggestion. I think it is easier to read.
```suggestion
    this.customAttributesToAnnotationAllowlist = options.customAttributesToAnnotationAllowlist || [];
```

Reply:
Done in: https://github.com/RoadieHQ/roadie-backstage-plugins/pull/908/commits/22d7bd33793c4fd5aab31bdde831c175701ca2d0